sample/fybrikapplication.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: app.fybrik.io/v1beta1
```

Comment:
Should it be part of the sample ?

Reply:
Yes, it's a sample fybrik application isn't it?